testnet/full-node.md

```diff
@@ -30,7 +30,7 @@ sudo rm -rf /usr/local/go
 1.2) Install latest/required Go version (installing `go1.17.2`)
 
 ```
-curl https://golang.org/dl/go1.17.2.linux-amd64.tar.gz
+curl -O https://dl.google.com/go/go1.17.2.linux-amd64.tar.gz
```

Comment:
Looks like this is behind a redirect. Updated it in `genesis-validators.md ` as well.